docs/config.rst

```diff
@@ -435,11 +436,38 @@ The algorithm used by trust manager factory for SSL connections. Default value i
   * Importance: low
 
 Configuration Options for SASL Authentication between REST Proxy and Apache Kafka Brokers
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------------------------------------------------------------------------------
+
+Kafka SASL configurations are described `here <http://docs.confluent.io/3.0.0/kafka/sasl.html>`_
 
 Note that all the SASL configurations (for REST Proxy to Broker communication) are prefixed with "client". If you want the configuration to apply just to consumers or just to producers, you can replace the prefix with "consumer" or "producer" respectively.
 
-In addition to these configurations, make sure ``bootstrap.servers`` configuration is set with SASL_PLAINTEXT://host:port (or SASL_SSL) end-points, or you'll accidentally open an SASL connection to a non-SASL port.
+In addition to these configurations, make sure ``bootstrap.servers`` configuration is set with SASL_PLAINTEXT://host:port (or SASL_SSL://host:port) end-points, or you'll accidentally open an SASL connection to a non-SASL port.
+
+Pass the name of the JAAS file, and the name of Kerberos config file via environment variables to each Kafka broker, Schema Registry, and REST Proxy. For example:
```

Comment:
I think this section and what follows is a bit confusing... 

First, we need to make clear that schema registry is optional, otherwise people freak out :)
Second, since we are mixing configuration for multiple components and they should have their own docs, maybe link to the security parts of their documentation. And make it clear that this is just a reminder (except for the REST Proxy part where this is the actual docs). 

If you have no idea what I want, we can merge this and I'll get add my own PR :)